Makefile

```diff
@@ -2,23 +2,33 @@ REGISTRY         ?= docker.io
 PROJECT          ?= ansibleplaybookbundle
 TAG              ?= latest
 BROKER_APB_IMAGE = $(REGISTRY)/$(PROJECT)/ansible-service-broker-apb
+BUILD_DIR        = "${GOPATH}/src/github.com/fusor/ansible-service-broker/build"
 
-build: $(shell find cmd pkg)
+install: $(shell find cmd pkg)
 	# HACK: Unless docker's vendor directory is removed, we end up with a
 	# duplicate symbol error from the linker that prevents compilation.
 	rm -rf ${GOPATH}/src/github.com/fusor/ansible-service-broker/vendor/github.com/docker/docker/vendor && \
-		go install ./cmd/broker
+	go install ./cmd/broker
```

Comment:
OK, this will conflict with my change. I'm willing to rebase this branch and get this merged.